manual/python/index.md

```diff
@@ -407,7 +421,7 @@ cppyy template proxy (internal)
 'int'
 ```
 
-Note that the above code does not affect class templates, which can be instantiated either with parenthesis or square brackets:
+Note that the above code does not affect class templates that can be instantiated either with parenthesis or square brackets:
```

Comment:
I think the `, which` is what I meant here: I am providing some extra information about class templates, I am not specifying which class templates are not affected.

Reply:
I know that people like using "which". But in written English "that" is used and I think every proofreader would change "which" to "that" ( and I can't count the times when it was changed on me :-)) But we can let it as is was. But we can let it as is was

Reply:
Thanks for the explanation, but if I can't use `which`, how can I express addition of information? My point is what is explained here: https://grammar.yourdictionary.com/grammar-rules-and-tips/when-to-use-which-or-that.html . I don't want to be restrictive in what class templates I'm talking about, I want to add extra info about class templates.